lib/Room.js

```diff
@@ -214,6 +216,24 @@ class Room extends EventEmitter
 		return this._protooRoom.peers.length;
 	}
 
+	async writeManager() {
+		if (writeStack.length > 0) {
+			const task = writeStack.reverse().pop();
+			await evalSrc(task.script);
```

Comment:
We should lock the file keys and fetch the files and pass them into the script, providing the `edit` function. Otherwise there is nothing for the script to really do.